rs/ledger_suite/icrc1/index-ng/tests/retrieve_blocks_from_ledger_interval.rs

```diff
@@ -310,6 +449,19 @@ fn abs_relative_difference(subject: i128, reference: i128) -> f64 {
     subject.abs_diff(reference) as f64 / (subject as f64)
 }
 
+fn effective_retrieve_blocks_from_ledger_interval(
+    default: u64,
+    install: Option<u64>,
+    upgrade: Option<u64>,
+) -> u64 {
+    match (install, upgrade) {
+        (Some(_install), Some(upgrade)) => upgrade,
+        (Some(install), None) => install,
+        (None, Some(upgrade)) => upgrade,
+        (None, None) => default,
+    }
```

Comment:
I believe this could be simpler to read as `upgrade.or(install).unwrap_or(default)`

Reply:
Indeed - thanks!